db.tf

```diff
@@ -0,0 +1,22 @@
+resource "random_password" "password" {
+  length           = 16
+  special          = true
+  override_special = "#$%"
+}
+resource "aws_db_instance" "database" {
+  allocated_storage   = 10
+  engine              = "mysql"
+  engine_version      = "8.0"
+  instance_class      = "db.t2.micro"
+  identifier          = "test-db-instance"
+  name                = "mydb"
+  username            = "admin"
+  password            = random_password.password.result
+  skip_final_snapshot = true
+
+  tags = {

```

Comment:
Things like `tags` are really annoying to deal with on every resource, so I suggest doing something like this --

```
locals {
  common_tags = {
    "Owner" = "pei.b.liu"
    "Environment" = "dev"
}
```

and then merging it with custom tags (like Name) on the individual resources.

```
tags = merge(local.common_tags, {
  "Name" = "terraform-eks-demo-db"
}
```